### ISSUE TITLE
Provide headers back to glibc2.5

### ISSUE THREAD

**Issue by LordAro** [wheybags/glibc_version_header#6]
For reasons that I promise mostly make sense, I have need to support libc versions back to RedHat/CentOS 5, which is glibc 2.5.

It'd be great if these could be provided. Some experimentation indicates that compiling glibc with modern GCC might be tricky (I was using 5.4), but some hacking using a COS5 chroot seemed to get a complete header.

**Comment by wheybags** [comment]
I'll try add support tomorrow. Given your other issue (#4), maybe a newer compiler version might work better? 2.13 compiles just fine for me without adding the flags you mentioned there using gcc 7.2.

**Comment by wheybags** [comment]
also
> reasons that I promise mostly make sense

curiosity, but would you care to elaborate? :p

**Comment by LordAro** [comment]
Excellent, thanks

Nothing really interesting, other than some people are still using CentOS5...

**Comment by wheybags** [comment]
Something to note, it seems gcc itself generates calls to some glibc functions, and doesn't use the designated symver. I've made a post about this on the gcc mailing list: https://gcc.gnu.org/ml/gcc/2018-01/msg00221.html
For now, that seems to limit the effectiveness of targeting anything before glibc 2.14 (the version that introduced the latest memcpy). I'm not sure what version is latest for the other symbols.
For rederence, I noticed this when building gcc itself :v
